bundles/org.openhab.binding.openthermgateway/README.md

```diff
@@ -71,7 +71,16 @@ The OpenTherm Gateway binding supports the following channels:
 | airpressfault             | Switch               | Air pressure fault                                       | R      |
 | waterovtemp               | Switch               | Water over-temperature fault                             | R      |
 | oemfaultcode              | Switch               | OEM fault code                                           | R      |
-| diag                      | Switch               | Diagr / wstics indication                                | R      |
+| diag                      | Switch               | Diagnostics indication                                   | R      |
+| unsuccessfulburnerstarts  | Number:Dimensionless | Unsuccessful burner starts                               | R      |
+| burnerstarts              | Number:Dimensionless | Burner starts                                            | R      |
+| chpumpstarts              | Number:Dimensionless | Central heating pump starts                              | R      |
+| dhwpvstarts               | Number:Dimensionless | Domestic hot water pump/valve starts                     | R      |
+| dhwburnerstarts           | Number:Dimensionless | Domestic hot water burner starts                         | R      |
+| burnerhours               | Number:Dimensionless | Burner hours                                             | R      |
+| chpumphours               | Number:Dimensionless | Central heating pump hours                               | R      |
+| dhwpvhours                | Number:Dimensionless | Domestic hot water pump/valve hours                      | R      |
+| dhwburnerhours            | Number:Dimensionless | Domestic hot water burner hours                          | R      |
```

Comment:
These sound rather `Number:Time`, than `Dimensionless`.

Reply:
Added Number:Time for these channels